content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,190 @@
+menu, and :guilabel:`Save` changes. Then, navigate back to the :guilabel:`Locations` overview, and
+repeat this step for the second physical :guilabel:`Warehouse` stock :guilabel:`Location`. Be sure
+to :guilabel:`Save` changes again.
 
-Sell a product from the virtual warehouse
-=========================================
+Both locations are now *child locations* of the virtual warehouse *parent location*. This allows
+stock to be taken from multiple locations to fulfill a single sales order, if there is not enough
+stock in any one location (provided they are both tied to the same virtual warehouse *parent
+location*).
 
-Let’s say you have two products, one stored in warehouse A and one stored in warehouse B. Now, you
-can create a new quotation for one of each product. Go to other information and choose Warehouse A+B
-in the shipping information.
+Example flow: Sell a product from a virtual warehouse
+=====================================================
 
-.. image:: stock_warehouses/ware3.png
+To sell products from multiple warehouses using a virtual *parent* location, there must be at least
+**two** products and at least **two** warehouses configured - with at least **one** product with
+quantity on-hand in each warehouse, respectively.
+
+To create a new Request for Quotation, or RFQ, navigate to the :menuselection:`Sales` app, and
+click :guilabel:`Create` from the :guilabel:`Quotations` overview. Fill out the information on the
+new quotation by adding a :guilabel:`Customer`, and click :guilabel:`Add a product` to add the two
+products stored in the two warehouses.
+
+Then, click the :guilabel:`Other Info` tab. Under the :guilabel:`Delivery` section, change the
+:guilabel:`Warehouse` listed to the virtual warehouse that was previously created. Once the
+warehouse has been changed, click :guilabel:`Confirm` to confirm the quotation and create a sales
+order.
+
+Now that the quotation has been confirmed as a sales order, click the :guilabel:`Delivery` smart
+button. From the :guilabel:`Warehouse Delivery form`, click :guilabel:`Edit`, and change the
+:guilabel:`Source Location` to the *Stock* location of the virtual warehouse that was previously
+created, it it was not already automatically set as the *source location*.
+
+.. important::
+   The :guilabel:`Source Location` and :guilabel:`Warehouse`, under the :guilabel:`Other Info` tab
+   on the :guilabel:`Sales Order form`, must match in order for the products included in the sales
+   order to be pulled from different warehouses.
+
+.. image:: stock_warehouses/stock-warehouses-delivery-order.png
    :align: center
+   :alt: Delivery order with matching source and child locations.
+
+Finally, from the :guilabel:`Warehouse Delivery form`, if they aren't set already, click
+:guilabel:`Edit` and select the drop-down menus to change the *Locations* under the
+:guilabel:`From` column for each product to the *child locations* previously tied to the virtual
+*parent* location.
```

Comment:
Paragraphs written this way are confusing to me.
- Where _exactly_ on the form?
- Give direct instructions, not "may or may not" hypotheticals 😉 

Note:
You may need to touch this section up later as the line items for this part of the delivery order did *not* automatically populate for me.

```suggestion
Finally, on the warehouse delivery form, under the :guilabel:`Detailed Operations` tab, confirm
that the *Locations* values under the :guilabel:`From` column for each product matches to the *child
locations* that are tied to the virtual *parent location*.
```